### ISSUE TITLE
There's an incorrect link in the doc of "HTTP Headers"

### ISSUE THREAD

**Issue by satansk** [akka/akka-http#1416]
In the doc of [HTTP Headers](http://doc.akka.io/docs/akka-http/current/scala/http/common/http-model.html#http-headers), there is a link that can't work properly:

```
See [this test] for a full table of what happens when.
```
The link of this test is `/akka-http-core/src/test/scala/akka/http/impl/engine/rendering/ResponseRendererSpec.scala#L422` , and I think `#L422` is unnecessary.

**Comment by jonas** [comment]
I see two problems: `#` is encoded causing a 404 error and the line number is out of date. It would be cool if we could either use an expandable `@@snip` or fix the `@github` directive to support linking to a snippet marker.

**Comment by satansk** [comment]
Thank you @jonas , I get it, `#L422` is indicating the line number, but I notice there is another link looks exactly like this one works fine, it's [here](http://doc.akka.io/docs/akka-http/10.0.10/scala/http/routing-dsl/directives/custom-directives.html#map-and-tmap):

```
One example of a predefined directive relying on map is the [optionalHeaderValue] directive.
```

**Comment by jonas** [comment]
OK, the other link was "fixed" in #900. However, I think it would be better to fix this upstream in Paradox to make the `@github` directive support line numbers.

**Comment by ktoso** [comment]
yeah, definitely - would you open a ticket on paradox about this please?

**Comment by jonas** [comment]
Opened the "best" kind of ticket: https://github.com/lightbend/paradox/pull/143

**Comment by pvlugter** [comment]
https://github.com/lightbend/paradox/pull/143 included in paradox `0.3.0`

**Comment by jonas** [comment]
Fixed in #1436
